src/channel/playlist.js

```diff
@@ -1046,7 +1052,7 @@ PlaylistModule.prototype._addItem = function (media, data, user, cb) {
         if (this.items.insertAfter(item, this.current.uid)) {
             if (existing && !allowDuplicates) {
                 item.temp = existing.temp;
-                this._delete(existing.uid);
+                this._delete(existing.uid, "[server]");
```

Comment:
I think it is weird to send a username in this case.

The socket frames sent by CyTube for playlist management are more like commands for the playlist UI rather than a log of actions initiated by some user.  In this particular case, the fact that it is implemented as a delete is an implementation detail -- logically what is happening is a move (this particular branch is when Queue Next is used on an item already on the playlist in a different position; assuming duplicates are not allowed, it moves the queued item to the front of the queue by inserting a new item there and removing the old one).

At various points in the past I have had ideas to improve the channel log to function more like a machine-readable audit log; I never got around to it before, but I feel like that would be a better solution to this type of use case (see who did what) by recording actions (and relevant session data) instead of the low level mechanics of how the UI should update in response to those actions.